bitrise.yml

```diff
@@ -3,115 +3,89 @@ default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
-        inputs:
-        - command: install
-    - path::./:
-        title: Step Test
-        run_if: true
-        inputs:
-        - project_path: $BITRISE_SOURCE_DIR
-        - expo_cli_verson: "latest"
-        - override_react_native_version: 0.55.4
-    
-  expo_kit:
+  eject:
     before_run:
-    - _clear_workdir
+      - _clear_workdir
     steps:
-    - script:
-        title: Clone sample app
-        inputs:
-        - content: git clone -b expo_sdk $SAMPLE_APP_URL . && cd
-    - npm@0.9.1:
-        title: Install node modules for the expo project
-        inputs:
-        - command: install
-    - path::./:
-        title: Step Test
-        run_if: true
-        inputs:
-        - project_path: $BITRISE_SOURCE_DIR
-        - expo_cli_verson: "latest"
-        - user_name: $USER_NAME
-        - password: $PASSWORD
+      - script:
+          title: Clone sample app
+          inputs:
+            - content: git clone $SAMPLE_APP_URL . -b $BRANCH && cd
+      - npm@0.9.1:
+          title: Install node modules for the expo project
+          inputs:
+            - command: install
+      - path::./:
+          title: Step Test
+          run_if: true
+          inputs:
+            - project_path: $BITRISE_SOURCE_DIR
+            - expo_cli_verson: "latest"
+            - override_react_native_version: 0.61.0
 
   _clear_workdir:
     envs:
     steps:
-    - script:
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-            cd ${ORIGIN_SOURCE_DIR}
-            rm -rf "./_tmp"
-            mkdir "_tmp"
-    - change-workdir:
-        title: Switch working dir to test / _tmp dir
-        description: |-
-          To prevent step testing issues, like referencing relative
-          files with just './some-file' in the step's code, which would
-          work for testing the step from this directory directly
-          but would break if the step is included in another `bitrise.yml`.
-        run_if: true
-        inputs:
-        - path: ${ORIGIN_SOURCE_DIR}/_tmp
-        - is_create_path: true
+      - script:
+          inputs:
+            - content: |-
+                #!/bin/bash
+                set -ex
+                cd ${ORIGIN_SOURCE_DIR}
+                rm -rf "./_tmp"
+                mkdir "_tmp"
+      - change-workdir:
+          title: Switch working dir to test / _tmp dir
+          description: |-
+            To prevent step testing issues, like referencing relative
+            files with just './some-file' in the step's code, which would
+            work for testing the step from this directory directly
+            but would break if the step is included in another `bitrise.yml`.
+          run_if: true
+          inputs:
+            - path: ${ORIGIN_SOURCE_DIR}/_tmp
+            - is_create_path: true
 
   dep-update:
     title: Dep update
     description: |
       Used for updating bitrise dependencies with dep
```

Comment:
Used for updating bitrise dependencies with dep > Update Bitrise dependencies with dep.

Reply:
As discussed over Slack this is not user facing.